src/nccl_ofi_rdma.c

```diff
@@ -3859,7 +3859,9 @@ static int accept(nccl_net_ofi_listen_comm_t *listen_comm,
  exit:
 
 	/* Close receive communicator in case listen operation failed */
-	ret = close_listen_recv_comm(l_comm);
+	if (close_listen_recv_comm(l_comm)) {
```

Comment:
But now if `ret` is success before the call to `close_listen_recv_comm`, you're ignoring any error in `close_listen_recv_comm`.